ludwig/utils/dataframe_utils.py

```diff
@@ -61,7 +61,12 @@ def to_numpy_dataset(df: DataFrame, backend: Optional["Backend"] = None) -> Dict
         res = df[col]
         if backend and is_dask_backend(backend):
             res = res.compute()
-        dataset[col] = np.stack(res.to_numpy())
+        try:
+            dataset[col] = np.stack(res.to_numpy())
+        except ValueError:
```

Comment:
can we check for the dataframe to be empty instead of triggering a ValueError?

Reply:
Good suggestion! Done.